core/modules/system/system.install

```diff
@@ -50,72 +50,120 @@ function system_requirements($phase) {
-      'severity' => REQUIREMENT_INFO,
-    );
-  }
+  // PHP info and memory usage.
+  $php_version = phpversion();
+  $description_items = array();
+  $php_fix_info = array();
+
+  // Set some defaults.
+  $requirements['php'] = array(
+    'title' => $t('PHP'),
+    'value' => $t('Version: !version', array('!version' => $php_version)),
+    'severity' => REQUIREMENT_INFO,
+  );
 
   // Minimum recommended PHP version can be incremented when deprecating older
-  // versions of PHP but while they are still supported. This does not throw
-  // a warning or error on updates to keep upgrade tests compatible.
+  // versions of PHP but while they are still supported. This does not throw a
+  // warning or error on updates to keep upgrade tests compatible.
   $minimum_recommended_version = '5.6.0';
   $minimum_warning = $t('Your PHP installation is too old. Backdrop requires at least PHP %version.', array('%version' => $minimum_recommended_version));
-  if (version_compare($phpversion, $minimum_recommended_version) < 0) {
+  if (version_compare($php_version, $minimum_recommended_version) < 0) {
     $requirements['php']['description'] = $minimum_warning . ' ' . l(t('Read about the minimum version increase'), 'https://backdropcms.org/news/backdrop-cms-drops-support-for-php-versions-prior-to-56') . '.';
-    $requirements['php']['severity'] = $phase === 'runtime' ? REQUIREMENT_ERROR : REQUIREMENT_INFO;
+    if ($phase === 'runtime') {
+      $requirements['php']['severity'] = REQUIREMENT_ERROR;
+    }
   }
 
   // If PHP does not meet the bare minimum, remaining checks are unnecessary.
-  if (version_compare($phpversion, BACKDROP_MINIMUM_PHP) < 0) {
+  if (version_compare($php_version, BACKDROP_MINIMUM_PHP) < 0) {
     $requirements['php']['description'] = $minimum_warning;
     $requirements['php']['severity'] = REQUIREMENT_ERROR;
     return $requirements;
   }
 
-  // Test PHP register_globals setting.
-  $requirements['php_register_globals'] = array(
-    'title' => $t('PHP register globals'),
+  // Append link to php info page if phpinfo() is available.
+  if (function_exists('phpinfo')) {
+    if ($phase == 'runtime') {
+      $requirements['php']['value'] .= ' (' . l(t('PHP information'), 'admin/reports/status/php') . ')';
+    }
+  }
+  else {
+    $requirements['php']['description'] = $t('The <code>phpinfo()</code> function has been disabled for security reasons. To see your server\'s <code>phpinfo()</code> information, change your PHP settings. For more information, see the <a href="@phpinfo">Enabling and disabling phpinfo()</a> handbook page.', array('@phpinfo' => 'https://backdropcms.org/user-guide/enabling-and-disabling-phpinfo'));
+  }
+
+  if ($phase === 'runtime') {
+    $php_mem_usage_info = $t('PHP memory usage: @usage | @peak (peak) | @true (true)', array(
+      '@usage' => format_size(memory_get_usage()),
```

Comment:
@klonos do you happen to know what memory_get_usage() measures?

I suppose it's the memory usage of the current process. Wouldn't that be the identical number every time? The amount of memory the status page needed?